content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -9,7 +9,7 @@ title: Persistent Volumes
 feature:
   title: Storage orchestration
   description: >
-    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as <a href="https://aws.amazon.com/products/storage/">AWS</a> or <a href="https://cloud.google.com/storage/">GCP</a>, or a network storage system such as NFS, iSCSI, Ceph, Cinder.
+    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider, or a network storage system such as iSCSI or NFS.
```

Comment:
It should be fine because we are providing pointers to some very generic portals from public cloud providers. This is not about whether the support to those storage is in-tree or not.

Reply:
Why is it useful to list AWS and GCP as examples, though? Does that help the reader?